deploy/sample-backup-configs/sample_config_hwdb.ini

```diff
@@ -8,6 +8,11 @@
 # Only AGFIs for the latest release of FireSim are guaranteed to be available.
 # If you are using an older version of FireSim, you will need to generate your
 # own images.
+[firerocketboom-1x1core-no-nic-ddr3-llc4mb]
```

Comment:
What is 1x1core

Reply:
+1

Reply:
It was my attempt at matching what the other designs do: aka instead of "dualcore" I put "1x1core" to mean "1Rx1B".

Reply:
Spoke with Sagar offline and he prefers `firerocketboom-1rx1b-no-nic-ddr3-llc4mb` or something like that.

Reply:
Will resolve in other PR